code/game/objects/items/weapons/switchtool.dm

```diff
@@ -126,12 +126,15 @@
 
 	if(!stored_modules[module])
 		return
+	if(deployed)
+		return
```

Comment:
null and 0 are also not equal, and if you're a complete idiot/mad genius, you can exploit that